tests/pytest_tests/system_tests/test_core/test_public_api.py

```diff
@@ -274,21 +274,20 @@ def test_run_queue(user):
 
 
 @pytest.mark.nexus_failure(feature="artifacts")
-def test_run_log_artifact(relay_server, wandb_init):
-    with relay_server():
```

Comment:
Why did you remove it? What server is the test talking to now?

Reply:
it will talk to the test container (see `wandb_init` fixture for details). You only need the relay_server if you want to test something related to intercepting the traffic between the client and server.